modules/installation-and-upgrade/pages/container-deployment/mlm/server-deployment-vm-mlm.adoc

```diff
@@ -24,9 +24,7 @@ Images for {productname} {productnumber} are available at link:https://www.suse.
 [IMPORTANT]
 ====
 Remember to resize the default size of the partition before deploying.
-For more information on preparing raw images, see:
-
-* link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation[]
+For more information on preparing raw images, see link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation[]
```

Comment:
I guess in master/5.1 we must link to (maybe, there are more links to fix later):
```suggestion
For more information on preparing raw images, see https://documentation.suse.com/sle-micro/6.1/html/Micro-deployment-raw-images-virtual-machines/index.html#deployment-preparing-configuration-device[].
// For the 5.0 backport:
// For more information on preparing raw images, see link:https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation[].
```